dts/bindings/arm/nordic,nrf21540-fem.yaml

```diff
@@ -0,0 +1,46 @@
+# Copyright (c) 2020 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+
+description: >
+    This is a representation of the nRF21540 Radio Front-End module
+compatible: "nordic,nrf21540-fem"
```

Comment:
> We shall postpone merging it until DTS format in [nrfconnect/sdk-nrf#2896](https://github.com/nrfconnect/sdk-nrf/pull/2896) is finalized.

This PR also defines a binding for the compatible "nordic,nrf21540-fem".

May I assume that PR will be updated to remove the binding, since it is going upstream?

Reply:
That PR needs to be aligned with this PR after all the changes, including removing unnecessary bindings